x-pack/plugins/infra/public/metrics_overview_fetchers.ts

```diff
@@ -77,22 +76,22 @@ export const combineNodeTimeseriesBy = (
 
 export const createMetricsFetchData = (
   getStartServices: InfraClientCoreSetup['getStartServices']
-) => async ({
-  startTime,
-  endTime,
-  bucketSize,
-}: FetchDataParams): Promise<MetricsFetchDataResponse> => {
+) => async ({ absoluteTime, bucketSize }: FetchDataParams): Promise<MetricsFetchDataResponse> => {
   const [coreServices] = await getStartServices();
   const { http } = coreServices;
+
+  const from = moment(absoluteTime.start).valueOf();
+  const to = moment(absoluteTime.end).valueOf();
```

Comment:
It seems like we have time in three different formats:
 - relative time: `now-24h`
 - absolute time: `2020-07-08T16:56:32.429Z`
 - epoch timestamp: `1594227392429`

Do we need both absolute and epoch?

Reply:
`APM` and `Uptime` use `absolute time as string` while  `Logs` and `Metrics` use `absolute time as number`

Reply:
Ah, okay. Makes sense. Looking at the APM internals it seems like we are actually also converting the `absolute time as string` to `absolute time as number`:
https://github.com/elastic/kibana/blob/8a1d7d7e407f4e62c52ca597911b8024a4006d19/x-pack/plugins/apm/server/lib/helpers/setup_request.ts#L103-L104

But for now the APM apis expect a string so we should probably leave it at that. 


Right now landing page supplies `absolute time as string` and then consumers (like Logs and Metrics) must do the conversion to number.
I think it would be better to flip it so Obs landing page supplies `absolute time as number` and the consumers will need to convert it to string (`new Date(1331209044000).toISOString();`).

I think that would be more standard and we would no longer have to convert from number to string for Elastic charts:
https://github.com/elastic/kibana/blob/429c39cf15f241c7cdd599987808e2c3f3584a15/x-pack/plugins/observability/public/components/app/section/apm/index.tsx#L50-L51

Reply:
This is no big deal, so if it's a bigger change you can just ignore it. Just an idea.

Reply:
That's also good for me.